dhis-2/pom.xml

```diff
@@ -707,18 +707,8 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-dependency-plugin</artifactId>
                     <version>${maven-dependency-plugin.version}</version>
-                    <executions>
```

Comment:
Are you removing it since we are now explicitly calling the plugin in the pipeline? It would still be good to automatically run it locally, so devs get notified earlier (locally) of dependency issues. Currently it would run with `mvn verify` or `mvn install` which I think makes sense.

Reply:
You are right, I put it back and I also removed the condition.
It didn't make sense to have one and the other, skipping with the label but then running it during install wouldn't make much sense.
We are now forcing the PR to be dependencies compliant